bokehjs/src/lib/models/widgets/tables/data_table.ts

```diff
@@ -255,6 +256,21 @@ export class DataTableView extends WidgetView {
       if (checkboxSelector != null)
         this.grid.registerPlugin(checkboxSelector)
 
+      if(!options.editable){
+        const pluginOptions = {
+          dataItemColumnValueExtractor: function(val: any, col: any) {
+            let value: any = val[col.field]
+            if ( typeof value === 'string'){
+              value = value.replace(/\n/g, "\\n")
+            }
+            return value
+          },
+          includeHeaderWhenCopying : false,
+        }
```

Comment:
This code needs a cleanup. Please fix whitespace around `(`, `{` and `:`. `typeof` should be replaced with `isString()` function (from `core/util/types`). `any` types need to be replaced with more specific types or, if there is no other better choice, there needs to be a comment explaining why `any` is being used, including the best approximation of a possible type.

Reply:
@mattpap I was unable get any output from tslint, and py.test -m codebase doesn't seem to test the typescript syntax. I could not find instructions on the "Getting Set Up" -guide. I tried to fix the issues by guessing what exactly you meant. Hopefully that is close enough.

Reply:
Argh. Line 260 still needs a space before `{`.